CHANGELOG.rst

```diff
@@ -14,6 +14,13 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+Modified
```

Comment:
Are you planning on adding [0.7.2]? Won't you need this released?

Reply:
I plan on holding off on a release until I get further in license-manager PR(https://github.com/openedx/license-manager/pull/384). I think I jumped the gun on last release and the design wasn't ready for a release.

Reply:
@dianakhuang:
1. Modified => Changed would make consistent with the rest of this changelog.
2. Bringing your attention to Jinder's release plan, which you may or may not want to follow as you see fit.

Reply:
Yup, I was thinking of waiting until we got further in this ticket to release a new events version.